code.test.js

```diff
@@ -8,6 +8,12 @@ const test =
         var a1 = JSON.parse(JSON.stringify(arr));
         var a2 = JSON.parse(JSON.stringify(arr));
         var count = permutationSort(a1);
-        return count >= 0 && JSON.stringify(a1) == JSON.stringify(a2.sort(function(a, b) { return a - b; }));
+
+        if(permutationSort(a2)) {
+            return JSON.stringify(a1) === JSON.stringify(a2);
```

Comment:
Don't understand why you added this.

Reply:
test.js can't pass because of a counter example of [0,0]. I changed it again, and run correctly on my computer.